prime-router/metadata/hl7_mapping/ADT_A01/ADT_A01-base.yml

```diff
@@ -0,0 +1,42 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ADT_A01
+constants:
+  # Prefix for RS custom extension URLs
+  rsext: '"https://reportstream.cdc.gov/fhir/StructureDefinition/"'
+elements:
+  - name: message-headers
+    condition: >
+      Bundle.entry.resource.ofType(MessageHeader).exists() and
+      Bundle.entry.resource.ofType(MessageHeader).event is Coding and
+      Bundle.entry.resource.ofType(MessageHeader).event.code = 'A01'
+    resource: 'Bundle.entry.resource.ofType(MessageHeader)'
+    required: true
+    schema: ../ORU_R01/base/message-header
+
+  - name: software-segment
+    condition: 'Bundle.entry.resource.ofType(MessageHeader).exists()'
+    resource: 'Bundle.entry.resource.ofType(MessageHeader)'
+    schema: ../ORU_R01/base/software
+
+  - name: patient-information
+    resource: 'Bundle.entry.resource.ofType(Patient)'
+    condition: '%resource.count() = 1'
+    required: true
+    constants:
+      hl7SegmentGroup: ''
+    schema: ../ORU_R01/base/patient
+
+  - name: patient-contact
+    resource: 'Bundle.entry.resource.ofType(Patient).contact'
+    condition: '%resource.exists()'
+    constants:
+      hl7SegmentGroup: ''
+    schema: ../ORU_R01/base/patient-contact
```

Comment:
Isn't this kind of weird that we depend on a different unrelated message type? Should we take the base patient-contact out of ORU-R01?

Reply:
Yes, there is another issue to clean this up. https://github.com/CDCgov/prime-reportstream/issues/8445